File: arrays/maxElement.cpp
#include <bits/stdc++.h> 
int largestElement(vector<int> &arr, int n) {
    int maxNum = arr[0];
    for(int i=0; i<n; i++) {
        if (maxNum < arr[i]) {
            maxNum = arr[i];
        }
    }
    return maxNum;
}

File: pythonArray/MaxElement.py
from sys import *
from collections import *
from math import *

def largestElement(arr: [], n: int) -> int:

    # Write your code from here.
    # max_num = arr[0]
    # for i in range(n):
    #     if (max_num < arr[i]):
    #         max_num = arr[i]
    
    arr.sort()

    return arr[-1]